### PR TITLE
fix: Handle missing fields

### LINKED ISSUE #29: Need to handle missing fields in the response
**Describe the bug**
All the fields displayed on `InfoActivity` are not always present in response from the API. For example, some users don't have `ratings` and hence the `maxRank`. This currently throws a `nullPointerException` and should be handled to avoid crashes.

**Expected Behaviour**

Respective fields should either be hidden or show something like 'N/A'.

**Steps to reproduce it**

Try any user handle who haven't yet solved a problem or participated in competition. You won't get rating in the response for such users. You can try 'the_vision'.

**LogCat for the issue**

```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.example.immadisairaj.codeforces, PID: 27969
    java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String java.lang.Integer.toString()' on a null object reference
        at com.example.immadisairaj.codeforces.InfoActivity.showInfo(InfoActivity.java:167)
        at com.example.immadisairaj.codeforces.InfoActivity$2.onResponse(InfoActivity.java:95)
        at retrofit2.ExecutorCallAdapterFactory$ExecutorCallbackCall$1$1.run(ExecutorCallAdapterFactory.java:70)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:216)
        at android.app.ActivityThread.main(ActivityThread.java:7285)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:975)
```

**Screenshots of the issue**

App immediately restarts.

**Smartphone Info:**

| Device          | Samsung M30s |
|-----------------|--------------|
| Android Version | Pie 9.0      |

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Kshitij09**
Fixes #29 

**Changes:**

Handled every field in the API call's response and handled missing field with defualt 'N/A' values. This guarantees that app will never crash again for this cause. If the user account exists on codeforces, it'll be shown in the app. I've made following changes:

1. Handled fields for null values and replaced them with default (N/A) strings to avoid app crash.
1. Updated strings.xml names as per the naming [convention.](https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md#2322-strings)
1. Used parameterized strings for `textview.setText()` instead of appending them programmatically.

Please make sure these boxes are checked before submitting your pull request - thank you!

- [X] Run unit tests with `./gradlew check` to make sure you didn't break anything

- [X] If you have multiple commits please combine them into a single commit by squashing them.

Please add screenshots/GIFs here if there are any changes to the UI below.

All Fields example
Fields empty example
Location Empty example

### CODE REVIEW

**Comment by Kshitij09** [comment]
Is there anything wrong with the commits included in this PR ?

**Comment by immadisairaj** [comment]
@Kshitij09 this PR also has the commits from #27(closed) which fixes #22. So, if you want to fix #22 then send a separate PR. This PR only has to fix #29. Hope you'll fix it soon. 👍

**Comment by immadisairaj** [comment]
> Please add screenshots/GIFs here if there are any changes to the UI below.
> 
> All Fields example
> Fields empty example
> Location Empty example

These screenshots have no image content. So, also make sure the images appear properly.

**Comment by Kshitij09** [comment]
**All Fields**

accessibility text

**No Rating**

accessibility text

**No Address, No Name**

accessibility text

File: app/src/main/java/com/example/immadisairaj/codeforces/Api/Info/Info.java [hunk 1]
```diff
@@ -31,5 +31,4 @@ public String getStatus() {
         return result;
     }
 
-
```
Comment (immadisairaj) [inline app/src/main/java/com/example/immadisairaj/codeforces/Api/Info/Info.java]: revert the whole file

**Review by immadisairaj — CHANGES_REQUESTED** [review]

**Comment by immadisairaj** [comment]
@Kshitij09 can you squash the commits into one

File: app/src/main/java/com/example/immadisairaj/codeforces/Api/Info/Info.java (hunk 1 above)
Comment (Kshitij09) [reply app/src/main/java/com/example/immadisairaj/codeforces/Api/Info/Info.java]: Okay.

**Comment by Kshitij09** [comment]
Yes, I'll do that soon

**Comment by Kshitij09** [comment]
> @Kshitij09 can you squash the commits into one

I guess it has to be done by you while merging the pull request ([reference](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges))
I can only squash all commits before creating a pull request.

File: app/src/main/java/com/example/immadisairaj/codeforces/Api/Info/Info.java (hunk 1 above)
Comment (Kshitij09) [reply app/src/main/java/com/example/immadisairaj/codeforces/Api/Info/Info.java]: Done.

**Comment by immadisairaj** [comment]
> > @Kshitij09 can you squash the commits into one
> 
> I guess it has to be done by you while merging the pull request ([reference](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges))
> I can only squash all commits before creating a pull request.

You can squash after creating a PR and then force push to the branch from which you are sending the PR.

And the file is not yet reverted. Can you check that once

File: app/src/main/java/com/example/immadisairaj/codeforces/Api/Info/Info.java (hunk 1 above)
Comment (immadisairaj) [reply app/src/main/java/com/example/immadisairaj/codeforces/Api/Info/Info.java]: It's not yet done. This file should not be included in the pr

**Review by immadisairaj — CHANGES_REQUESTED** [review]

**Review by immadisairaj — APPROVED** [review]

**Pull request merged**